PROJECT

```diff
@@ -19,7 +18,6 @@ resources:
   version: v1alpha1
 - api:
     crdVersion: v1
-    namespaced: false
```

Comment:
is the removal of these intentional?

Reply:
just how kubebuilder did it. Doesn't have an affect on the code or crds so it's just a default thing for them